azure-cs-aks-helm/MyStack.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
+
+using Pulumi;
+using Pulumi.Kubernetes.Core.V1;
+using Pulumi.Kubernetes.Helm;
+using Pulumi.Kubernetes.Helm.V3;
+
+class MyStack : Stack
+{
+
+    [Output]
+    public Output<string> ApacheServiceIP { get; set; }
+
+    [Output]
+    public Output<string> ClusterName { get; set; }
+
+    [Output]
+    public Output<string> Kubeconfig { get; set; }
+
+    public MyStack()
+    {
+        var myConfig = new MyConfig();
+        var myCluster = new MyCluster(myConfig);
+
+        var chart = new Chart("apache-chart", new ChartArgs() {
+			Chart = "apache",
```

Comment:
Here I originally had `Repo = "bitnami"` as an arg to ChartArgs, as in the other examples, but was getting failure. This was a tad confusing, especially since I couldn't figure out which kubernetes plugin version it opts into - maybe this was not seeing my fix in https://github.com/pulumi/pulumi-kubernetes/pull/1491  that got released as pulumi-kubernetes 2.8.3.

What is better for the example anyway, maybe we should remove Repo = "bitnami" from the rest of example  too?